Watchman.Discord/Areas/Help/Factories/HelpInformationFactory.cs

```diff
@@ -17,7 +17,13 @@ public HelpInformationFactory(ArgumentInfoFactory argumentInfoFactory)
         public HelpInformation Create(BotCommandInformation commandInformation)
         {
             var argumentInformations = commandInformation.BotCommandArgumentInformations.Select(x => this._argumentInfoFactory.Create(x));
-            var descriptions = new List<Description> { new Description() };
+            var desciptionCommand = typeof(Devscord.DiscordFramework.Framework.Commands.BotCommandDescription.CommandDescription).GetProperties()
+                .Where(x => x.PropertyType.Name == "String" && x.Name == commandInformation.Name)
+                .Select(prop =>
+                { 
+                    return prop.GetValue(prop)?.ToString();
+                }).FirstOrDefault();
+            var descriptions = string.IsNullOrEmpty(desciptionCommand) ? new List<Description>() : new List<Description> { new Description() { Language = "EN", Text = desciptionCommand } };
```

Comment:
nową liste tworzysz zawsze
możesz zrobić tak że jeśli `!string.IsNullOrEmpty(desciptionCommand)` to dodajesz element 

+co jeśli informacja o komendzie już istnieje w bazie? z tego co kojarze wtedy informacja nie będzie zaktualizowana, tylko zignorowana
to zadziała tylko na pustej bazie

na 90%, nie jestem pewny

Reply:
Ta właśnie się zastanawiałem czy iść w stronę by aktualizowało wpis o opisie czy raczej to się wyzeruje i odpali by wygenerować komendy z opisami. Będę poprawiał szefie 👍

Reply:
no tylko czy wywalanie całej listy z bazy po każdej drobnej aktualizacji ma sens?